module/modmanager/manager.go

```diff
@@ -335,6 +417,28 @@ func (m *module) startProcess(ctx context.Context, parentAddr string, logger gol
 	return nil
 }
 
+func (m *module) stopProcess() error {
+	if m.process == nil {
+		return nil
+	}
+	defer utils.UncheckedErrorFunc(func() error {
+		// Attempt to remove module's .sock file if module did not remove it
+		// already.
+		if _, err := os.Stat(m.addr); err == nil {
+			return os.Remove(m.addr)
+		}
+		return nil
+	})
+
+	// Stop managed process and swallow 143 errors. Some stopped modules may return
+	// "exit status 143" indicating a graceful exit after SIGTERM.
+	if err := m.process.Stop(); err != nil &&
+		!strings.Contains(err.Error(), gracefullyExitedMessage) {
```

Comment:
This [Python SDK PR](https://github.com/viamrobotics/viam-python-sdk/pull/250) modified how Python modules stop to be slightly more graceful. They return `exit status 143`, which is indicative of a graceful shutdown after reception of a `SIGTERM`. We could add this error swallowing to `goutils`, too.

Reply:
So it's a long, long established standard that 0 is "exited normally" in linux/unix process space. Why are we using a strange custom code? This feels very wrong to me, and nothing in the PR or the linked jira issue mentions anything about defining custom exit codes or behavior.

Reply:
Can you let me know how to reproduce this? I pulled down the latest python sdk, went into the module in the examples directory, ran `poetry install` and then `./run.sh /tmp/test.sock` and it started up using poetry, no issues. I then sent it a SIGTERM with `kill -TERM $PID` (with the actual pid.) It exited gracefully, and the exit code (determined with `echo $?`) was simply 0.

If I forced a kill with -KILL then I got code 137 (which is 128+ the signal (kill==9) that it died on.) That typically means a signal that is NOT handled correctly. 143 (128+15) would, I believe mean something similar, that things died because of a SIGTERM (15) signal, but the shutdown was NOT graceful/error-free.